docs/pages/blog/2021.md

```diff
@@ -0,0 +1,185 @@
+
+What's our mission? We're making building great UIs more accessible.
+
+2021 was great, 2022 is going to be even more exciting!
+We will continue in the same direction.
+Here is a breakdown of our [roadmap](/discover-more/roadmap/).
+
+### MUI Core
+
+The release of v5 has introduced a significant API churn in the community.
+While our [versioning frequency](/versions/#release-frequency) aims for one major per year, we hope we can iterate on v5 during the whole year of 2022 without any breaking changes, e.g. taking full advantage of the new style engine.
+
+#### MUI Base
+
+To continue improving the customization experience, we will double down on the version of the components without styles: `@mui/base`.
+These components and hooks contain the main functionalities and accessibility, without being opinionated about how styles are applied or what styles.
+We still have work to do to have a full set of unstyled components.
+
+You can read more about them in [the docs](/customization/unstyled-components/) and keep track of our progress in [#27170](https://github.com/mui-org/material-ui/issues/27170).
+
+Our high-level plan is to use the unstyled components and hooks as the basis of the Material Design components and our second design system.
+
+#### MUI System
+
+There a number of improvements that we can bring to the system (the npm package `@mui/system` used to create the styled components).
+Hopefully, we will bring native support for CSS variables, improve DX, and improve performance.
+
+#### MUI Material
+
+Google is rolling out [Material Design 3](https://m3.material.io/), we have to determine how to respond to it.
+Should we implement it later on for v6? Should we ignore it? Should we implement Google's UI that looks cleaner instead?
+In the meantime, we will keep improving the components, increasingly relying on `@mui/base`.
+
+#### MUI Joy
+
+We will continue [in the direction of 2021](/blog/2020/#a-second-theme).
+While Material Design is a predominant design choice for building rich client-side applications, it's not the only popular design trend.
+We plan to provide a [second design system](https://github.com/mui-org/material-ui/issues/22485) to expand the range of use cases for MUI, striking a balance between:
+
+- alignment with the best-looking design trends of hot startups in the US.
+- optimization of its usage for rich client-side applications while keeping it good enough for landing pages.
+- simplicity of the design for allowing simpler customizations.
+- complementarity with Material Design.
+
+It will be built on top of the unstyled components: MUI Base and our styling library: MUI System.
+
+### MUI X
+
+The data grid product initiative is working, it's time to scale it by growing the team and the product scope.
+We will keep the data grid as our primary effort, there are still [many features left to be built 🚧](/components/data-grid/getting-started/#feature-comparison) to get closer to the alternatives.
```

Comment:
Does it make sense to create a separate title only for the DataGrid, like "MUI Material"? We could mention the opening of the apiRef on the MIT version, full documentation, hook-only API and the Premium plan.